src/main/scala/com/fulcrumgenomics/umi/CallMolecularConsensusReads.scala

```diff
@@ -114,6 +124,7 @@ class CallMolecularConsensusReads
       minInputBaseQuality          = minInputBaseQuality,
       minConsensusBaseQuality      = minConsensusBaseQuality,
       minReads                     = minReads,
+      producePerBaseTags           = outputPerBaseTags,
```

Comment:
I'm on the fence.  I worded them differently because one _produces_ tags and the other _outputs_ them to a file on disk.